OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp

```diff
@@ -978,6 +978,16 @@ void SimbodyEngine::convertRadiansToDegrees(Storage &rStorage) const
     scaleRotationalDofColumns(rStorage, SimTK_RADIAN_TO_DEGREE);
     rStorage.setInDegrees(true);
 }
+
+void SimbodyEngine::convertRadiansToDegrees(TimeSeriesTable& table) const {
+    constexpr double PI{3.141592653589793};
+    constexpr double RAD_TO_DEG{180 / PI};
+
+    table.updMatrix() *= RAD_TO_DEG;
+    table.removeTableMetaDataKey("inDegrees");
+    table.addTableMetaData("inDegrees", std::string{"yes"});
```

Comment:
I believe @chrisdembia is referring to `convertRadiansToDegrees(Storage &)` vs this implementation for the table. This method is incorrectly converting the entire table to degrees (and is using a different constant - should also use `SimTK_RADIAN_TO_DEGREE` here. `SimbodyEngine::scaleRotationalDofColumns()` only converts columns that are associated with rotational coordinates.